fire/core.py

```diff
@@ -566,8 +569,13 @@ def _ParseFn(args):
       # If we're allowed *varargs or **kwargs, there's always capacity.
       capacity = True
 
-    if fn_keywords is None and kwargs:
-      raise FireError('Unexpected kwargs present', kwargs)
+    extra_kw = set(kwargs) - set(fn_kwonly)
+    if fn_keywords is None and extra_kw:
+      raise FireError('Unexpected kwargs present:', extra_kw)
+
+    missing_kwonly = set(required_kwonly) - set(kwargs)
+    if missing_kwonly:
+      raise FireError('Missing required flags:', missing_kwonly)
```

Comment:
Do you plan to keep this error printing to the terminal when a user mistypes something on the command line? Someone using the final command line program might be familiar with command line flags, but not the python specific concept of keyword only arguments.